src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2569,6 +2569,8 @@ because xbuild doesn't support framework reference assemblies.
 	<Delete Files="$(_AndroidMainDexListFile)" />
 	<Delete Files="$(_AndroidBuildIdFile)" />
 	<Delete Files="$(_ResolvedUserAssembliesHashFile)" />
+	<Delete Files="$(_AndroidCompileBindingJavaFileList)" />
+	<Delete Files="$(_AndroidCompileJavaFileList)" />
```

Comment:
If these are in `FileWrites`, do we need these here?

Sometimes, I find a list of files to manually delete, and they were only needed because they *weren't* in `FileWrites`.